chat-plugins/wifi.js

```diff
@@ -383,24 +383,24 @@ var commands = {
 	},
 	'': 'help',
 	help: function (target, room, user) {
-		if (room.id !== 'wifi') return this.sendReply("This command can be used only in the Wi-Fi room.");
+		if (room.id !== 'wifi') return this.sendReply('This command can only be used in the Wi-Fi room.');
 		if (!this.canBroadcast()) return;
 		this.sendReplyBox(
 			'<strong><em><font size="2" color="red"><center>Wi-Fi Room Giveaway help</center></font></strong></em><br />' +
 			'Note that all commands start with \'/giveaway\', with the exception of /gag.<br /><br />' +
 			'<strong>Player commands:</strong><br />' +
 			'- guess <small>or</small> /gag <em>answer</em> - Guesses the answer for a question giveaway<br />' +
-			'- viewanswer <small>or</small> answer - shows the answer in a question giveaway. (only to giveaway host/giver)<br />' +
-			'- remind - shows the details of a giveaway. (can be broadcast.)<br />' +
-			'- join <small>or</small> joinlottery - Joins a lottery giveaway.<br />' +
-			'- leave <small>or</small> leavelottery - Leaves a lottery giveaway.<br />' +
+			'- viewanswer <small>or</small> answer - shows the answer in a question giveaway (only to giveaway host/giver)<br />' +
+			'- remind - shows the details of a giveaway (can be broadcast)<br />' +
+			'- join <small>or</small> joinlottery - Joins a lottery giveaway<br />' +
+			'- leave <small>or</small> leavelottery - Leaves a lottery giveaway<br />' +
 			'<br />' +
 			'<strong>Staff commands:</strong><br />' +
-			'- question <small>or</small> qg <em>User, Prize, Question, Answer</em> - Start a new question giveaway (Requires: % @ #).<br />' +
-			'- lottery <small>or</small> lg <em>User, Prize, Number of Winners (optional)</em> - Starts a lotto giveaway. The number of winners is set to 1 by default. (Requires: % @ # & ~)<br />' +
-			'- changequestion - Changes the question of a question giveaway. Can only be done before the giveaway starts. (Requires: % @ #)<br />' +
+			'- question <small>or</small> qg <em>User, Prize, Question, Answer</em> - Start a new question giveaway. Requires: % @ #<br />' +
+			'- lottery <small>or</small> lg <em>User, Prize, Number of Winners (optional)</em> - Starts a lotto giveaway. The number of winners is set to 1 by default. Requires: % @ # & ~<br />' +
+			'- changequestion - Changes the question of a question giveaway. Can only be done before the giveaway starts. Requires: % @ #<br />' +
 			'- changeanswer - Changes the answer of a question giveaway. (Requires: % @ #)<br />' +
-			'- end - Forcibly ends the current giveaway (Requires: % @ #)<br />'
+			'- end - Forcibly ends the current giveaway. Requires: % @ #<br />'
```

Comment:
This is on purpose. All the help commands display the required ranks in this format. If you are proposing changing it, change it everywhere and in its own pull request.